app/decosjoin_service.py

```diff
@@ -211,15 +223,23 @@ def transform(self, zaken_source, user_identifier):  # noqa: C901
 
         deferred_zaken.sort(key=lambda x: x[0].get("caseType"))
 
+        # In parallel
         # Makes it possible to defer adding the zaak to the zaken response for example to:
         # - Adding dateWorkflowActive by querying other Api's
-        for [deferred_zaak, Zaak_cls] in deferred_zaken:
-            Zaak_cls.defer_transform(
+        def perform_deferred_transform(zaak_tuple):
+            [deferred_zaak, Zaak_cls] = zaak_tuple
+            return Zaak_cls.defer_transform(
                 zaak_deferred=deferred_zaak,
-                zaken_all=new_zaken,
+                zaken_all=[],
```

Comment:
Kun je dit er niet beter uithalen? Het lijkt erop dat dit niet meer gebruikt wordt in de Zaken.